src/routes/userRoutes.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import userController from '../controllers/userController';
+
+const router = express.Router()
+
+
+router.post('/register', userController.register)
+
+module.exports = router;
```

Comment:
Newline here or better still add `files.insertFinalNewline: true` to your VSCode `settings.json` file